posthog/settings.py

```diff
@@ -93,6 +93,9 @@ def get_env(key):
     "social_django",
 ]
 
+if DEBUG:
```

Comment:
Ah, this was the reason we didn't merge this last time, because some people have DEBUG running in production (for testing!) but won't have django_extensions installed. I think this snippet should be

```
try:
    import django_extensions
    INSTALLED_APPS.append("django_extensions")
except ImportError:
    pass
```

Reply:
:thinking: Great catch! Will update this soon.